### PR TITLE
Isomorphisms in large categories and large precategories

### PR DESCRIPTION
**Pull request by EgbertRijke**
This PR adds a file for isomoprhisms in large categories. Furthermore, it cleans up and refactors the file about isomorphisms in large precategories, and it also adds the theorems that a morphism is an isomorphism if and only if pre/postcomposition is an equivalence.

### CODE REVIEW

File: src/category-theory/large-precategories.lagda.md [hunk 1]
```diff
@@ -119,6 +119,26 @@ module _
...
+
+  postcomp-hom-Large-Precategory :
+    {l1 l2 l3 : Level}
+    (X : obj-Large-Precategory C l1)
+    {Y : obj-Large-Precategory C l2}
+    {Z : obj-Large-Precategory C l3}
+    (f : type-hom-Large-Precategory Y Z) →
+    type-hom-Large-Precategory X Y → type-hom-Large-Precategory X Z
+  postcomp-hom-Large-Precategory X f g =
+    comp-hom-Large-Precategory C f g
```
Comment (fredrik-bakke) [inline src/category-theory/large-precategories.lagda.md:141]: I put these under their own sections for small (pre)categories

**Comment by fredrik-bakke** [comment]
Damn, too slow 😞

File: src/category-theory/large-precategories.lagda.md (hunk 1 above)
Comment (EgbertRijke) [reply src/category-theory/large-precategories.lagda.md:141]: Ah, ok, perhaps you can sneak that into your categories PR:)

Comment (EgbertRijke) [reply src/category-theory/large-precategories.lagda.md:141]: Thanks for taking a look!

**Comment by EgbertRijke** [comment]
Sorry!

**Pull request merged**